includes/rest-api/class-sensei-rest-api-extensions-controller.php

```diff
@@ -135,6 +135,25 @@ function ( $plugin ) {
 		);
 	}
 
+	/**
+	 * Check user permission for reading plugins.
+	 *
+	 * @param WP_REST_Request $request WordPress request object.
+	 *
+	 * @return bool|WP_Error Whether the user can read extensions.
+	 */
+	public function can_user_read_plugins( WP_REST_Request $request ) {
```

Comment:
We can tighten this back up once we start loading the Extensions store only when we need it on the frontend.

Reply:
Just confirming. It's here just to catch the error when it's a teacher accessing the Home because they don't have the `activate_plugins` and `update_plugins`, right? And when we do what you said about not calling the Extensions store when not needed, this wouldn't be needed anymore?

Reply:
Correct and correct!